### PR TITLE
Fixes (with workaround) hashicorp-vault test running localy. Relates to #7543

### PR DESCRIPTION
**Pull request by JiriOndrusek**
Workaround for https://github.com/apache/camel-quarkus/issues/7543

 When hashicorp test runs on the system, which contains this module's /target/classes folder, test  fails.
 Reason is explained in the ticket https://github.com/apache/camel-quarkus/issues/7543

 Artifact created by this module does not contain anything (except license) in the target/classes folder.
 Workaround loads and installs  the licences from the different folder (target/license) and therefore does not trigger the Quarkus classloader.

### CODE REVIEW

**Comment by JiriOndrusek** [comment]
@jamesnetherton I think that https://github.com/apache/camel-quarkus/issues/7543 can be closed, but follow-up ticket would be nice (to solve the problem "cleanly" - this is more like a workaround)

**Review by gansheer — APPROVED** [review]

File: extensions-support/spring/shade/core/pom.xml [hunk 1]
```diff
@@ -82,6 +82,35 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Workaround for https://github.com/apache/camel-quarkus/issues/7543
+                 See more information in th ticket.
+                 Shortly, when hashicorp test runs onb the system, which contains this module/target/classes, it fails.
+                 Reason is explained in the ticker https://github.com/apache/camel-quarkus/issues/7543
+                 Artifact created by this module does mot contain anything (except license) in the target/classes folder.
+                 Workaround loads and installs  the licences from the folder target/license
+                 -->
```
Comment (jamesnetherton) [inline extensions-support/spring/shade/core/pom.xml]:
```suggestion
            <!--
                 Workaround for https://github.com/apache/camel-quarkus/issues/7543.

                 This module produces an artifact that does not include any compiled classes only license files, which causes tests to fail when the empty target/classes directory is detected.

                 The workaround explicitly loads and installs the license files from the target/license directory.
            -->
```

**Review by jamesnetherton — APPROVED** [review]
Added some suggested wording tidy-ups.

File: extensions-support/spring/shade/core/pom.xml (hunk 1 above)
Comment (JiriOndrusek) [reply extensions-support/spring/shade/core/pom.xml]: Thanks for the rewording my comment! It's fixed in the PR now.
